config/application.rb

```diff
@@ -19,6 +19,8 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
+    config.time_zone = "London"
```

Comment:
What about `config.time_zone = "UTC"`? as won't London change between `GMT` and `BST`

Reply:
That's what we're trying to achieve 👍 The current time in UTC is 1 hour behind BST, so if someone scheduled a publish for 10am on Tuesday it would actually be 9am UTC, which isn't the behaviour they want.

Setting to "London" accounts for both GMT and BST, so can schedule times predictably.